openapi/components/schemas/CouponRestrictionExclusiveAppliance.yaml

```
@@ -0,0 +1,12 @@
+type: object
+description: |-
+  Determines whether a coupon may be used in combination with other coupons. 
+  A coupon with this restriction enabled is applied only if it gives a bigger discount than the other (non-exclusive) coupon(s) together. 
+  If those conditions are met and this coupon is applied, all other coupons are removed.
```

suggestion: I'm not sure if this sentence is needed
(It seems to be repeating the first line a bit)
```suggestion
  If a coupon with this restriction is applied, all other coupons are removed.
```